lib/restAPIs/library.js

```diff
@@ -0,0 +1,47 @@
+const assert = require("assert");
+
+const assertParam = require("../util/assertParam");
+const RestUtil = require("../util/RestUtil");
+const buildCommonQuery = require("../util/buildCommonQueryParams");
+
+module.exports = class RestTopic extends RestUtil {
+  _getBase() {
+    return "library";
+  }
+
+  getLibrary(fields = "", outputFlag = "DEFAULT") {
```

Comment:
- Add basic JSDoc on method summary and params please. 
- Since `fields` is a general yet optional var included with most APIs, I would prefer the order to be 
```
getLibrary(outputFlag, fields) { } 
```
It's more likely someone will override the output flag than the fields param.

Reply:
Done.